src/main/scala/api/commands/igblastn.scala

```diff
@@ -0,0 +1,121 @@
+    (organism.type := organism.Raw)            ::
+    (clonotype_out.type := clonotype_out.Raw)  ::
+    (out.type := out.Raw)                      ::
+    *[AnyDenotation]
+
+  type OptionsVals =
+    (num_threads.type         := num_threads.Raw)         ::
+    (evalue.type              := evalue.Raw)              ::
+    (max_target_seqs.type     := max_target_seqs.Raw)     ::
+    (strand.type              := strand.Raw)              ::
+    (word_size.type           := word_size.Raw)           ::
+    (show_gis.type            := show_gis.Raw)            ::
+    (ungapped.type            := ungapped.Raw)            ::
+    (penalty.type             := penalty.Raw)             ::
+    (reward.type              := reward.Raw)              ::
+    (perc_identity.type       := perc_identity.Raw)       ::
+    (num_alignments_V.type    := num_alignments_V.Raw)    ::
+    (num_alignments_D.type    := num_alignments_D.Raw)    ::
+    (num_alignments_J.type    := num_alignments_J.Raw)    ::
+    (min_V_length.type        := min_V_length.Raw)        ::
+    (min_J_length.type        := min_J_length.Raw)        ::
+    (min_D_match.type         := min_D_match.Raw)         ::
+    (D_penalty.type           := D_penalty.Raw)           ::
+    (extend_align5end.type    := extend_align5end.Raw)    ::
+    (num_clonotype.type       := num_clonotype.Raw)       ::
+    (show_translation.type    := show_translation.Raw)    ::
+    (domain_system.type       := domain_system.Raw)       ::
+    (focus_on_V_segment.type  := focus_on_V_segment.Raw)  ::
+    *[AnyDenotation]
+
+  /* Default values match those documented in [the official BLAST docs](http://www.ncbi.nlm.nih.gov/books/NBK279675/) whenever possible. */
+  val defaults: Options := OptionsVals = options (
+    num_threads(1)                          ::
+    evalue(BigDecimal(10))                  ::
+    max_target_seqs(500)                    ::
+    strand(Strands.both)                    ::
+    word_size(11)                           ::
+    show_gis(false)                         ::
+    ungapped(false)                         ::
+    penalty(-3)                             ::
+    reward(2)                               ::
+    perc_identity(0D)                       ::
+    num_alignments_V(3)                     ::
+    num_alignments_D(3)                     ::
+    num_alignments_J(3)                     ::
+    min_V_length(9)                         ::
+    min_J_length(0)                         ::
+    min_D_match(5)                          ::
+    D_penalty(-4)                           ::
+    extend_align5end(false)                 ::
+    num_clonotype(100)                      ::
+    show_translation(false)                 ::
+    domain_system(IgBLASTDomainSystem.imgt) ::
+    focus_on_V_segment(false)               ::
+    *[AnyDenotation]
+  )
+
+  /* `igblastn` does not support the csv output format */
+  type ValidOutputFields =
+    |[AnyOutputField]
```

Comment:
Same as the others; output is just a *file*, the value of the `out` arg. We will need to override/change the toSeq mechanism for expressions: we want a different output format for IgBLAST commands. Better to make it a typeclass.